.github/CODEOWNERS

```diff
@@ -58,14 +58,14 @@
 # .NET Client Tools
 ###########
 /src/dotnet/                            @azure/azsdknet @jsquire
-/src/dotnet/APIView/                    @chidozieononiwu @praveenkuttappan @helen229
+/src/dotnet/APIView/                    @tjprescott @AlitzelMendez @helen229 @praveenkuttappan @chidozieononiwu
```

Comment:
AlitzelMendez  and helen229 needs a public `Microsoft` membership.    Non-blocking, but would you please ask them to flip the private to public?